### DISCUSSION TITLE
blog/ambient-mesh-l7-traffic-path/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#316] [Announcements]
# blog/ambient-mesh-l7-traffic-path/

本文以图示和实际操作的形式详细介绍了 Ambient Mesh 中的七层（L7）流量路径。

http://localhost:1313/blog/ambient-mesh-l7-traffic-path/

**Comment by BrunoGao** [comment]
添加了分享本文功能

**Reply by rootsongjc** [reply]
你说的是代码复制按钮吗？
